python/plugins/observables/pyscf_observable.py

```diff
@@ -35,6 +35,7 @@ def fromOptions(self, inputParams):
         mol = gto.mole.Mole()
         mol.atom = inputParams['geometry']
         mol.basis = inputParams['basis']
+        mol.build()
```

Comment:
what happened here? why did you need to add this method call? @danclaudino is this needed?

Reply:
`build()` needs to explicitly called only when changing members of `gto.mole.Mole()`, which is not the case here

Reply:
Without that line, you get a warning about calling scf before initializing the molecule. It's found in https://sunqm.github.io/pyscf/_modules/pyscf/scf/hf.html, specifically in the SCF class. In there they have an example of how to use the class, and a conditional to add that that mol.build line if it was not already present.

    def __init__(self, mol):
        if not mol._built:
            sys.stderr.write('Warning: %s must be initialized before calling SCF.\n'
                             'Initialize %s in %s\n' % (mol, mol, self))
            mol.build()

Reply:
I just tried and did get a warning. It may be because of a newer version, but does not impact performance or change the expected behavior in anyway, as it does call `build()`. Maybe we can accept this to avoid having this warning show up.

Reply:
cool I vote to accept it

Reply:
I'm cool with it